docs/RFCs/private-network.md

```diff
@@ -0,0 +1,66 @@
+- Feature Name: private-network
+- Start Date: 2022-02-01
+- RFC PR:
+- Functionland Issue:
+
+## Background
+we are using IPFS as our file system. but IPFS is build to use for public data, and it does not support ACL, 
```

Comment:
@gitaaron 
I don't think the security layer that cover all the stories would be ready in at least 3 month. and also i personally can't relay on access control by encryption only to make user safe. with this solution we can demo useful box and photo in in around a week.
As @masih mention i should explain more on public network risk's

Reply:
@farhoud  thanks for the clarification.  I am still not sure where this PR is coming from (probably because I joined the team after this work commenced).  Could you please share a few of the use cases / user stories this PR is solving?  Thanks!

Reply:
It seems to me that this PR is trying to cover two different scenarios - 

1.  Private communication between the fula-client and fula-api (eg/ uploading a photo to a BOX and retrieving a photo from a BOX)
2. Private communication between BOXes or other 'peers' in the fula-network (eg/ data syncing)

Is  that correct?

Reply:
yes. and i try to add more on it.